packages/react/readme.md

```diff
@@ -108,7 +108,7 @@ const { address, isConnected } = useAccount()
 
 // Returns
 interface Return {
-  address?: string
+  address: string | ''
```

Comment:
Yeh, but I wanted to highlight it in the docs, that address can be an empty string if user is not connected instead of undefined. Reason for it not being undefined is to make composability with hooks like `useEnsName` `useBalance` a bit cleaner (they expect string for some required options, however if this string is empty or there are missing required options, they handle it internally). Get's rid of patterns like this

```ts
const { data, error, isLoading, refetch } = useBalance({
    addressOrName: address ?? '',
    enabled: Boolean(address)
 })
```